optimism.tokenlist.json

```diff
@@ -1199,11 +1199,33 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+    {
+      "chainId": 1,
+      "address": "0x03e173ad8d1581a4802d3b532ace27a62c5b81dc",
+      "name": "Thales",
```

Comment:
Set the name to match L1: `Thales Token`

Reply:
Changed it to `Optimistic Thales Token` to match the L1 contract. Also posted the same change with the L2 token.